messages/format.go

```diff
@@ -0,0 +1,48 @@
+package messages
+
+import "fmt"
+
+// FormatAmount formats a cent amount as dollars.
+func FormatAmount(v int64) string {
+	f := "-$%d.%02d"
```

Comment:
```suggestion
	f := "$%d.%02d"
```

Should the format string be initialised without the minus sign first?

Reply:
Good catch.